bionic/persistence.py

```diff
@@ -412,6 +412,20 @@ class MetadataMatch:
     level = attr.ib()
 
 
+# TODO Should we merge this with InventoryEntry?
```

Comment:
This makes sense to me. Maybe phrase it as an action item instead of a question.

Reply:
I could go either way, since we may want to add a more expensive field to one but not the other, and it's easier to merge them than un-merge them.